### PR TITLE
Add lambda

### PR DESCRIPTION
**Pull request by lefado**
I have added a function `batch_lambda_run()` into `src/autoscale_aws/batch_lambda.py`. This function will zip all the python project located into the folder `src/autoscale_aws/aws`.

- Python main file to be run on lambda, needs to be named `main.py`. Main function of this file needs to be named `lambda_handler()`

Travis seems not happy about something. It is unrelated to my development

### CODE REVIEW

File: src/autoscale_aws/util_aws.py [hunk 1]
```diff
@@ -1537,6 +1538,47 @@ def sftp_walk(remotepath):
...
+        aws_access_key_id=aws_access,
+        aws_secret_access_key=aws_secret,
+        region= aws_region)
+    
+
+    function_name = f'lambda_from_util_aws'
+    # Create lambda
+    zip = open('src/autoscale_aws/aws/lambda.zip', 'rb')
+    # runtime options: nodejs'|'nodejs4.3'|'nodejs6.10'|'nodejs8.10'|'nodejs10.x'|'nodejs12.x'|'nodejs14.x'|'java8'|'java8.al2'|'java11'|'python2.7'|'python3.6'|'python3.7'|'python3.8'|'python3.9'|'dotnetcore1.0'|'dotnetcore2.0'|'dotnetcore2.1'|'dotnetcore3.1'|'nodejs4.3-edge'|'go1.x'|'ruby2.5'|'ruby2.7'|'provided'|'provided.al2'
+    runtime = 'python3.9'
```
Comment (lefado) [inline src/autoscale_aws/util_aws.py:1570]: I have set it to use Python 3.9. This can be modified

File: src/autoscale_aws/util_aws.py [hunk 2]
```diff
@@ -1537,6 +1538,47 @@ def sftp_walk(remotepath):
...
+    role = 'arn:aws:iam::495134704719:role/lambda_from_util_aws'
```
Comment (lefado) [inline src/autoscale_aws/util_aws.py:1571]: This need to be changed to an IAM role arn. A new role should be created, givint to the role full access to: S3, Lambda, and CloudWatch.

File: src/autoscale_aws/util_aws.py (hunk 1 above)
Comment (arita37) [reply src/autoscale_aws/util_aws.py:1570]: Please use python 3.7

File: src/autoscale_aws/util_aws.py (hunk 2 above)
Comment (arita37) [reply src/autoscale_aws/util_aws.py:1571]: Please put as comment in the code.
#### to be changed to an IAM role arn. Arole should be created, givint to the role full access to: S3,

**Pull request merged**